src/views/pages/customer/orders/single.twig

```diff
@@ -411,10 +412,30 @@
                             {% endif %}
                         </div>
                     {% endfor %}
-
                 </div>
             {% endfor %}
 
+            {% if order.options|length %}
+              <div class="bg-white p-5 rounded-md">
+                <h2 class="mb-3.5 text-sm font-bold">{{ trans('pages.cart.cart_options') }}</h2>
+                <div class="flow-root rounded-md px-4 border border-gray-200 mb-2">
+                  <dl class="text-sm text-dark divide-y divide-border-color">
+                    {% for item in order.options %}
+                      {% for option in item.options %}
+                        <div class="py-2.5 center-between">
+                          <dt class="mb-2 md:mb-0">
+                            {{ option.name }} {{'(' ~ item.amounts.total.amount|money ~ ')'}}:
+                          </dt>
+                          <dd class="font-medium text-xs">    
+                            {{ option.value.value }}
```

Comment:
why not `option.value`

Reply:
Because it returns from BE in multi level objects `{...option, value: {type: '', value: ''}}`